tests/JsonApiErrorMiddlewareTest.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+namespace WtfPhp\JsonApiErrors\Tests;
+
+use Exception;
+use PHPUnit\Framework\TestCase;
+use Psr\Http\Message\ResponseFactoryInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use WtfPhp\JsonApiErrors\JsonApiErrorMiddleware;
+use WtfPhp\JsonApiErrors\JsonApiException;
+use WtfPhp\JsonApiErrors\Tests\Fakes\TestRequest;
+use WtfPhp\JsonApiErrors\Tests\Fakes\TestResponseFactory;
+
+class JsonApiErrorMiddlewareTest extends TestCase
+{
+    protected ServerRequestInterface $request;
+
+    protected ResponseFactoryInterface $responseFactory;
+
+    protected JsonApiErrorMiddleware $middleware;
+
+    protected function setUp(): void
+    {
+        $this->request = new TestRequest();
+        $this->responseFactory = new TestResponseFactory();
+        $this->middleware = new JsonApiErrorMiddleware($this->responseFactory);
+    }
+
+    /** @test */
+    public function itShoudHandleAnExceptionWithoutErrorCodeAndMessage()
+    {
+        $nextHandler = new class implements RequestHandlerInterface {
+            public function handle(ServerRequestInterface $request): ResponseInterface
+            {
+                throw new Exception();
+            }
+        };
+
+        $response = $this->middleware->process($this->request, $nextHandler);
+
+        $this->assertEquals(500, $response->getStatusCode());
+        $this->assertEquals('An exception occured', $response->getReasonPhrase());
```

Comment:
`occurred` is written with two r's 🙂 
Can you exchange all occurrences of that word?